python/pyspark/pandas/groupby.py

```diff
@@ -44,10 +44,16 @@
     TYPE_CHECKING,
 )
 
-import numpy as np
 import pandas as pd
 from pandas.api.types import is_hashable, is_list_like
 
+if parse_version(pd.__version__) >= parse_version("1.3.0"):
```

Comment:
Shall we use `LooseVersion`?

Reply:
Yeah, I think using `LooseVersion` is consistent in the code base.

Reply:
Hi @Cedric-Magnan, do you have time to address this? Thanks.